modules/travel_pay/app/services/travel_pay/session.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class Session
```

Comment:
If we go with the auth_manager (vs. authenticated session) this file might be unnecessary?